src/flutter/shell/platform/linux_embedded/window/native_window_drm.cc

```diff
@@ -0,0 +1,175 @@
+// Copyright 2021 Sony Corporation. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/shell/platform/linux_embedded/window/native_window_drm.h"
+
+#include <fcntl.h>
+#include <xf86drm.h>
+
+#include <unordered_map>
+
+#include "flutter/shell/platform/linux_embedded/logger.h"
+#include "flutter/shell/platform/linux_embedded/surface/cursor_data.h"
+
+namespace flutter {
+
+NativeWindowDrm::NativeWindowDrm(const char* device_filename) {
+  drm_device_ = open(device_filename, O_RDWR | O_CLOEXEC);
```

Comment:
Close `drm_device_` in NativeWindowDrm's destructor instead of derived classes. open/close should be called and managed by only this class to avoid code duplication.

Reply:
Yes, I moved that into the destructor of supper class.

Reply:
missing `#include <unistd.h>`

Reply:
I've already fixed that.

Reply:
Thank you for your fix, but other members(e.g. `drm_crtc_`) have same problem. Could you fix these in next PR?